### PR TITLE
support transitive extension in TM definitions

### PR DESCRIPTION
**Pull request by sebastiankb**
solves #1374

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/pull/1478.html" title="Last updated on May 4, 2022, 3:48 PM UTC (d997484)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/1478/8baf584...d997484.html" title="Last updated on May 4, 2022, 3:48 PM UTC (d997484)">Diff</a>

### CODE REVIEW

File: index.template.html [hunk 1]
```diff
@@ -5239,8 +5239,13 @@ <h3>Extension and Import</h3>
     </pre>
 
     <p>
+        It is mentioned that the <code>tm:ref</code> or <code>tm:extends</code> features also allows transitive extension (a hierarchy of extensions). For example, assuming there are 3 TMs 
+        <code>ColorableLamp</code> which defines a <code>tm:extends</code> of the TM <code>DimmableLamp</code> which itself defines a <code>tm:extends</code> of the TM <code>SwitchableLamp</code>.
+        Consequently, the <code>ColorableLamp</code> TM also extend all definitions of both <code>DimmableLamp</code> and <code>SwitchableLamp</code>. It should be ensured that no recursive extensions are defined
```
Comment (thjaeckle) [inline index.template.html:5244]: Shouldn't the formulation "It should be ensured that no recursive extensions are defined" be instead:
"Recursive extensions leading to an infinite loop MUST NOT be defined."?

**Review by thjaeckle — CHANGES_REQUESTED** [review]
Remarks see my inline comments..

File: index.html [hunk 2]
```diff
@@ -6417,10 +6417,12 @@ <h3>Extension and Import</h3>
     </pre>
 
     <p>
-        It is mentioned that the <code>tm:ref</code> or <code>tm:extends</code> features also allows transitive extension (a hierarchy of extensions). For example, assuming there are 3 TMs 
-        <code>ColorableLamp</code> which defines a <code>tm:extends</code> of the TM <code>DimmableLamp</code> which itself defines a <code>tm:extends</code> of the TM <code>SwitchableLamp</code>.
-        Consequently, the <code>ColorableLamp</code> TM also extend all definitions of both <code>DimmableLamp</code> and <code>SwitchableLamp</code>. It should be ensured that no recursive extensions are defined 
-        that lead to an infinite loop. 
+        The <code>tm:extends</code> and the import mechanism based on <code>tm:ref</code> explicitly support transitive extension (a hierarchy of extensions).
```
Comment (thjaeckle) [inline index.html:6420]: "explicitly support" -> "explicitly support**s**"

File: index.html [hunk 3]
```diff
@@ -6417,10 +6417,12 @@ <h3>Extension and Import</h3>
     </pre>
 
     <p>
-        It is mentioned that the <code>tm:ref</code> or <code>tm:extends</code> features also allows transitive extension (a hierarchy of extensions). For example, assuming there are 3 TMs 
-        <code>ColorableLamp</code> which defines a <code>tm:extends</code> of the TM <code>DimmableLamp</code> which itself defines a <code>tm:extends</code> of the TM <code>SwitchableLamp</code>.
-        Consequently, the <code>ColorableLamp</code> TM also extend all definitions of both <code>DimmableLamp</code> and <code>SwitchableLamp</code>. It should be ensured that no recursive extensions are defined 
-        that lead to an infinite loop. 
+        The <code>tm:extends</code> and the import mechanism based on <code>tm:ref</code> explicitly support transitive extension (a hierarchy of extensions). 
+        For example, assuming there are 3 TMs: "A" which defines a <code>tm:extends</code> of the TM "B" which itself defines a <code>tm:extends</code> of the TM "C".
+        Consequently, the "A" TM also extend all definitions of both "B" and "C".
```
Comment (thjaeckle) [inline index.html:6422]: "also extend" -> "extend**s**"

**Review by thjaeckle — CHANGES_REQUESTED** [review]
addressed 2 little findings in inline comments

**Comment by sebastiankb** [comment]
from today's TD call, decided to merge

**Pull request merged**